src/node_debug_options.h

```diff
@@ -26,8 +29,9 @@ class DebugOptions {
  private:
 #if HAVE_INSPECTOR
   bool inspector_enabled_;
+  bool debug_brk_used_;
 #endif  // HAVE_INSPECTOR
-  bool wait_connect_;
+  bool wait_connect_;  // --inspect-brk
```

Comment:
If a comment is helpful here I think it could be a bit clearer - `wait_connect_` means to break before executing user code (i.e. on first line) and is used with both `--debug-brk` and `--inspect-brk`.